azure/Guidelines.md

```diff
@@ -760,79 +760,67 @@ while the Microsoft guidelines use the name "Operation-Location".
+:white_check_mark: **DO** use managed identity and Role Based Access Control ([RBAC](https://docs.microsoft.com/en-us/azure/role-based-access-control/overview)) as the mechanism allowing customers to grant permission to their Storage account to your service.
 
 :white_check_mark: **DO** Add RBAC roles for every service operation that requires accessing Storage scoped to the exact permissions.
 
 :white_check_mark: **DO** Ensure that RBAC roles are backward compatible, and specifically, do not take away permissions from a role that would break the operation of the service. Any change of RBAC roles that results in a change of the service behavior is considered a breaking change.
 
-##### Handling 'downstream' errors
-It is not uncommon to rely on other services, e.g. storage, when implementing your service. Inevitably, the services you depend on will fail. In these situations, you can include the downstream error code and text in the inner-error of the response body. This provides a consistent pattern for handling errors in the services you depend upon.
-
-:white_check_mark: **DO** include error from downstream services as the 'inner-error' section of the response body.
-
-#### Working with files
-Generally speaking, there are two patterns that you will encounter when working with files; single file access, and file collections.
-
-##### Single file access
-Desiging an API for accessing a single file, depending on your scenario, is relatively straight forward.
-
-:heavy_check_mark: **YOU MAY** use a Shared Access Signature [SAS](https://docs.microsoft.com/en-us/azure/storage/common/storage-sas-overview) to provide access to a single file. SAS is considered the minimum security for files and can be used in lieu of, or in addition to, RBAC.
+:heavy_check_mark: **YOU MAY** use Shared Access Signatures [SAS](https://docs.microsoft.com/en-us/azure/storage/common/storage-sas-overview) to provide access to Storage objects.
 
 :ballot_box_with_check: **YOU SHOULD** if using HTTP (not HTTPS) document to users that all information is sent over the wire in clear text.
 
-:ballot_box_with_check: **YOU SHOULD** support managed identity using Azure Storage by default (if using Azure services).
+:white_check_mark: **DO** return an HTTP status code representing the result of your service operation's behavior. If the operation uses Storage internally, this an implementation detail from the client's perspective of the operation but may happen due to a missing Storage object or insufficient permissions. If the internal Storage operation fails, include the Storage error information in the 'inner-error' section of your operation's response body to help the client know that they may have configured their Storage improperly. 
```

Comment:
I think we can make this a little more concise:
```suggestion
:white_check_mark: **DO** include the Storage error information in the `inner-error` section of an error response if the error was the result of an an internal Storage operation failure.  This may help the client determine the underlying cause of the error, e.g a missing Storage object or insufficient permissions.
```

Reply:
```suggestion
:white_check_mark: **DO** return an HTTP status code representing the result of your service operation's behavior. 

:white_check_mark: **DO** include the Storage error information in the 'inner-error' section of an error response if the error was the result of an internal Storage operation failure. This helps the client determine the underlying cause of the error, e.g.: a missing storage object or insufficient permissions. 
```